Source/Core/RowType.swift

```diff
@@ -202,7 +202,7 @@ extension RowType where Self: BaseRow {
      */
     @discardableResult
     public func onChange(_ callback: @escaping (Self) -> Void) -> Self {
-        callbackOnChange = { [unowned self] in callback(self) }
+        callbackOnChange = { [weak self] in callback(self!) }
```

Comment:
force unwrawpping a weak self is just as bad as unowned self.  My suggestion is test the optional properly.

`[weak self] in  guard let `self` = self else { return }; callback(self)`